models/action.go

```diff
@@ -345,12 +350,17 @@ func GetFeeds(ctx context.Context, opts GetFeedsOptions) (ActionList, error) {
 	opts.SetDefaultValues()
 	sess = db.SetSessionPagination(sess, &opts)
 
-	actions := make([]*Action, 0, opts.PageSize)
-
-	if err := sess.Desc("`action`.created_unix").Find(&actions); err != nil {
+	extActions := make([]*ExtAction, 0, opts.PageSize)
+	if err := sess.Desc("`action`.created_unix").Find(&extActions); err != nil {
 		return nil, fmt.Errorf("Find: %v", err)
 	}
 
+	actions := make(ActionList, 0, len(extActions))
+	for i := 0; i < len(extActions); i++ {
```

Comment:
Can't we use a simple for ... := range ?

Reply:
Why we should do that? In fact, the current implementation is safe when you want to change the content of the slice. Because `range` will have a copy of the element but this method will not.